lib/ecmascript.ts

```diff
@@ -4717,17 +4685,18 @@ export function RoundDuration(
   const TemporalDuration = GetIntrinsic('%Temporal.Duration%');
   let calendar, zdtRelative;
   // if it's a ZDT, it will be reassigned to a PDT below.
```

Comment:
This comment is inaccurate now

Reply:
Oh, you merged it already!  I'll submit a new PR to fix this up.

Reply:
Oops - the GH UI didn't show this to me :| That would be great.